LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2021 Gavin
+Copyright (c) 2022 Pattern Inc.
```

Comment:
This seems so impersonal...after all, Gavin has done so much for the company.

Reply:
I do ❤️ @gavinritchie